routes/web.php

```diff
@@ -75,12 +76,16 @@
         ]);
 })->name('logout');
 
+// Orders
+Route::get('orders/{category?}/{size?}', [OrderController::class, 'overview'])
+    ->name('home')
```

Comment:
Doordat je deze route home noemt krijg je een error in de routes, omdat er nu 2 dezelfde named routes zijn.